src/components/doc-type-tag.jsx

```diff
@@ -0,0 +1,33 @@
+import { classNames } from "@/utils/strings";
+
+export default function DocTypeTag(type) {
+	if (!type || !type.type) {
+		return;
+	}
+
+	const theType = type.type || type;
```

Comment:
It seems weird that the type could be nested one level (`type.type`) or it could just be the top-level `type`. Is there a way to make the format the same, or are they necessarily different?